satellite/admin.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellite
+
+import (
+	"bytes"
+	"context"
+	"encoding/hex"
+
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/pkg/macaroon"
+	"storj.io/storj/satellite/console"
+)
+
+// AdminConfig represents configuration for admin-bypass-type operations
+type AdminConfig struct {
```

Comment:
This should probably belong to some other package. `satellite` package is intended only for service wiring.